pkg/bbr/handlers/request.go

```diff
@@ -68,6 +78,48 @@ func (s *Server) HandleRequestBody(ctx context.Context, requestBodyBytes []byte)
 
 	metrics.RecordSuccessCounter()
 
+	//Mutate model name if it contains reserved keyword "lora" indicating that the requested model is lora (served from the same vLLM as the base model and from the same inferencepool)
+	//Convention: [model-family]/<model-name>/lora/<lora-name>
+	//Model name definition (the vLLM side) does not change: <my-arbitrary-lora-name>
+	//Model name in request (the client side): lora-name (no change from before)
+	loraTag := os.Getenv("LORA_TAG") //set via environment
+	if loraTag == "" {
+		loraTag = "lora"
+	}
+
+	orig := []byte(requestBody.Model)
+	prefix := []byte(requestBody.Model)
+	var suffix []byte
+
+	logger.V(logutil.DEFAULT).Info("Orig: " + string(orig))
+
+	if idx := bytes.Index(orig, []byte(loraTag)); idx != -1 {
+		lastSlash := bytes.LastIndex(orig[:idx], []byte("/"))
```

Comment:
is this assuming adapter name doesn't contain `/`?

Reply:
No, I take everything before `loraTag` as `prefix` and everything after `loraTag` as `suffix`.